modules/python3-pycrypto.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "python3-pycrypto",
+    "buildsystem": "simple",
+    "build-commands": [
+        "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"pycrypto\" --no-build-isolation"
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "http://vagueentertainment.com/tmp_pkg/pycrypto-2.6.1.tar.xz",
```

Comment:
this is already installed with python3-hivepy, I don't think you need this one here.

Reply:
This is a slightly modified version of pycrypto-2.6.1 with a small patch applied. I've put in a PR to fix the issue in the official release but as of writing it hasn't been applied.